src/components/TimeRemaining.js

```diff
@@ -11,8 +11,15 @@ export default function TimeRemaining({ activities }) {
       {remainingHours(activities) === 1 ? 'hour' : 'hours'} to add to your day,
       use {remainingHours(activities) === 1 ? 'it' : 'them'} wisely!
     </TimeLeftStyled>
```

Comment:
Imho this would be easier to read and more efficient if you calculate `remainingHours(` upfront and use the result here instead of calling it over and over again. Maybe extract the conditionals to their own functions as well for clarity?